src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -37,6 +38,19 @@ public UserDto createUser(CreateUserRequest request) {
         return UserMapper.toDto(user);
     }
 
+    public UserDto deleteUser(DeleteUserRequest request) {
+        User user = repository.findUserByNickname(request.getNickname());
+
+        if (user == null) {
+            throw new ApiException(UserError.NOT_FOUND);
+        }
+
+        UserDto ret = UserMapper.toDto(user);
```

Comment:
nit: I think you can delete this line and change the return to just be `return UserMapper.toDto(user)`